charts/fluentd/values.yaml

```diff
@@ -50,6 +53,47 @@ resources:
 
 nodeSelector: {}
 
+output:
```

Comment:
I would much better prefer to see a place where you could edit the various .conf files.

Makes it much more flexible.
I find this output option old and outdated and many times confusing

Having the section with the various conf files in the configmap and comment the examples/files that you dont use